### PR TITLE
Update to Readme

### PR DESCRIPTION
**Pull request by sawft99**
Updated readme organization, formatting, and grammar. Added a small amount of information. Removed unclear or redundant info. Provided more links/resources.

### CODE REVIEW

**Comment by sawft99** [comment]
Having never done any kind of RFID projects or worked with these devices, I wrote this from a newbie perspective. I had to do a great deal of digging around through other repositories and asking people questions to get started. A number of steps and information seem to be assumed. Others don't even add up for me. For example, when doing the automatic Windows install the batch file would not have worked if I didn't take the dfu zip from icemans repository. I'm not sure if the bootloader generation scripts in either repository would have addressed that in some way. I didn't really touch on that other thank linking a few things.

I also included  information, other tools, and how to use them from sources outside of this repository. I have a Tiny Pro and followed the directions for the automatic install on Windows section I wrote. I am not sure how accurate the Mini would be, but I think it should be correct. I replaced relevant buttons and much of the old documentation across the internet was made for it originally.

**Comment by sawft99** [comment]
I know noticed some of this, like dfu programmer location, is mentioned in the doxygen folder, but perhaps snippets would still be helpful on the front page readme

File: README.md [hunk 1]
```diff
@@ -1,142 +1,274 @@
...
+         - Open [Doc/Doxyfile](Doc/Doxyfile) and then in application `Show HTML Output`
+   - Documentation (emsec): 
+      - Doxygen: **[Here](https://rawgit.com/emsec/ChameleonMini/master/Doc/Doxygen/html)**
+      - Source: **[Here](/../../../../emsec/ChameleonMini/tree/master/Doc)**
+- Tools:
+   - Phone APP:
+      - Android: **[Chameleon](https://play.google.com/store/apps/details?id=com.proxgrind.chameleon)**
+         - The Android APP is currently **NOT** compatible with the official firmware
+      - IOS: **[ChameleonTiny Manager](https://apps.apple.com/us/app/chameleontiny-manager/id1601341565)**
+         - The IOS APP is currently **NOT** compatible with the official firmware
```
Comment (bettse) [inline README.md:26]: It isn't?

Comment (sawft99) [reply README.md:26]: I know I didn't pull that out of my butt. Can't currently find where I got that from though. I sourced from multiple places. Will attempt to find it again.

**Pull request merged**